From 02e915273845b21af223e7e5e2425569afcf1b83 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 28 Feb 2013 07:44:00 -0500
Subject: [PATCH] Fix DND with frame sync

Mutter magically ignores override-redirect windows with geometry
-100-100+1+1, and this breaks the frame synchronization between
gtk+ and mutter. For now, we avoid the issue by simply giving
the window a different geometry.
https://bugzilla.gnome.org/show_bug.cgi?id=694217
---
 gtk/gtkdnd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkdnd.c b/gtk/gtkdnd.c
index 42f7221..4b974a5 100644
--- a/gtk/gtkdnd.c
+++ b/gtk/gtkdnd.c
@@ -360,7 +360,7 @@ gtk_drag_get_ipc_widget_for_screen (GdkScreen *screen)
       result = gtk_window_new (GTK_WINDOW_POPUP);
       gtk_window_set_screen (GTK_WINDOW (result), screen);
       gtk_window_resize (GTK_WINDOW (result), 1, 1);
-      gtk_window_move (GTK_WINDOW (result), -100, -100);
+      gtk_window_move (GTK_WINDOW (result), -99, -99);
       gtk_widget_show (result);
     }  
 
-- 
1.7.10.4

